CMakeLists.txt

```diff
@@ -377,11 +380,10 @@ if (OPM_ENABLE_PYTHON)
 
   # -------------------------------------------------------------------------
   # 2: Embed the Python interpreter for keywords like PYACTION and PYINPUT
+  add_subdirectory(python/pybind11)
+  target_include_directories(opmcommon SYSTEM PRIVATE "python/pybind11/include;${PYTHON_INCLUDE_DIRS}")
   if (OPM_ENABLE_EMBEDDED_PYTHON)
-    add_subdirectory(python/pybind11)
-    target_include_directories(opmcommon SYSTEM PRIVATE "python/pybind11/include;${PYTHON_INCLUDE_DIRS}")
```

Comment:
Does this mean that everyone will always build the Python bindings?

Reply:
No - the diff you are looking at is still inside a 
```
if (OPM_ENABLE_PYTHON)
   ....
endif()
```

However; the keywordgenerator will create one file: `${BUILD}/tmp_gen/builtin_pybind.cpp` unconditionally, and if you are not doing Python it will not be used.